bulk/jobs.go

```diff
@@ -0,0 +1,126 @@
+package bulk
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	sfdc "github.com/g8rswimmer/go-sfdc"
+	"github.com/g8rswimmer/go-sfdc/session"
+)
+
+// Parameters to query the bulk jobs.
+//
+// IsPkChunkingEnabled will filter jobs with PK chunking enabled.
+//
+// JobType will filter jobs based on job type.
```

Comment:
Where does the filtering occur?

Reply:
It is the filtering for the get all jobs API